tests/test_z_cmdline.py

```diff
@@ -643,10 +643,12 @@ def test_hello(pytestconfig):
 
 def _alwayscopy_not_supported():
     # This is due to virtualenv bugs with alwayscopy in some platforms
-    # see: https://github.com/pypa/virtualenv/issues/565
+    # See: https://github.com/pypa/virtualenv/issues/565
```

Comment:
This fix is unrelated to this feature, should probably go into another bugfix pull request

Reply:
Okay I'll do that.  I thought that having it in a separate commit was adequate.